lib/blacklight/solr/search_builder_behavior.rb

```diff
@@ -52,30 +52,12 @@ def add_query_to_solr(solr_parameters)
       # solr LocalParams in config, using solr LocalParams syntax.
       # http://wiki.apache.org/solr/LocalParams
       ##
-      if search_field && search_field.solr_local_parameters.present?
-        local_params = search_field.solr_local_parameters.map do |key, val|
-          key.to_s + "=" + solr_param_quote(val, quote: "'")
-        end.join(" ")
-        solr_parameters[:q] = "{!#{local_params}}#{blacklight_params[:q]}"
-
-        ##
-        # Set Solr spellcheck.q to be original user-entered query, without
-        # our local params, otherwise it'll try and spellcheck the local
-        # params!
-        solr_parameters["spellcheck.q"] ||= blacklight_params[:q]
+      if search_field&.query_builder.present?
```

Comment:
This refactoring is great! 👏

Reply:
🤞 ; if nothing else, hopefully it unlocks the "new" JSON query DSL for us.